website/templates/log_list.mako

```diff
@@ -46,12 +46,17 @@
                                 <span class="contributor-anonymous">A user</span>
                             </span>
                                 <span data-bind="ifnot:log.anonymous">
+                                    <span data-bind="if: log.user.registered">
                                     <span data-bind="if: log.userURL">
                                         <a class="overflow" data-bind="text: log.userFullName || log.apiKey, attr: {href: log.userURL}"></a>
                                     </span>
                                     <span data-bind="ifnot: log.userURL">
                                         <span class="overflow" data-bind="text: log.userFullName"></span>
                                     </span>
+                                    </span>
+                                    <span data-bind="ifnot: log.user.registered">
+                                        Deactivated User
```

Comment:
This should be sentence-cased, as in the anonymized view above:

`A deactivated user [...]`

Also, this only works for logs that the deactivated user is the creator of, not the target of:

<img width="471" alt="screen shot 2016-05-23 at 11 22 56" src="https://cloud.githubusercontent.com/assets/5659262/15475854/0d63baf6-20da-11e6-8404-af6bc32bf648.png">

It's unclear whether or not this is even in the spec, so consider either removing or fixing, for consistency.